messaging/src/main/java/org/folio/spring/messaging/config/KafkaMessageConfig.java

```diff
@@ -0,0 +1,52 @@
+package org.folio.spring.messaging.config;
+
+import static org.apache.kafka.clients.consumer.ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG;
+import static org.apache.kafka.clients.consumer.ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.folio.spring.messaging.model.Event;
+import org.springframework.boot.autoconfigure.kafka.KafkaProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.kafka.annotation.EnableKafka;
+import org.springframework.kafka.config.ConcurrentKafkaListenerContainerFactory;
+import org.springframework.kafka.core.ConsumerFactory;
+import org.springframework.kafka.core.DefaultKafkaConsumerFactory;
+import org.springframework.kafka.support.serializer.JsonDeserializer;
+
+@EnableKafka
+@Configuration
+@Profile("messaging")
+public class KafkaMessageConfig {
+
+  // cut and past from - https://github.com/folio-org/mod-remote-storage/blob/master/src/main/java/org/folio/rs/config/KafkaConfiguration.java
```

Comment:
```suggestion
```
The "Cut and paste" should be documented in the commit.
I see no reason to have that in the code as it clutters it up.

Reply:
Removed.